app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: "careeratlas1@gmail.com"
```

Comment:
Do we have this as an ENV var? Might be better to only have to change the email in one place.